build.xml

```diff
@@ -28,15 +28,12 @@
   <property name="compile.java.bootclasspath" value="${build.bootclasspath}"/>
 
   <path id="adaptor.build.classpath">
-<!--
```

Comment:
Was this expecting the developer to dump the JDBC jar file in lib? Are there other jar file in lib we might need? Is this a shortcut for lib/**/*.jar?